sphinx/ext/inheritance_diagram.py

```diff
@@ -306,9 +306,10 @@ def generate_dot(self, name: str, urls: dict[str, str] | None = None,
             n_attrs.update(env.config.inheritance_node_attrs)
             e_attrs.update(env.config.inheritance_edge_attrs)
 
-        res: list[str] = []
-        res.append('digraph %s {\n' % name)
-        res.append(self._format_graph_attrs(g_attrs))
+        res: list[str] = [
+            'digraph %s {\n' % name,
```

Comment:
```suggestion
            f'digraph {name} {{\n',
```

Reply:
I was thinking of changing residual traditional string interpolation and format() to f-strings in a different PR, but perhaps there aren't so many of them.